frontend/src/layouts/EmptyLayout.tsx

```diff
@@ -0,0 +1,29 @@
+import './PublicLayout.scss';
+
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import { Footer, EmptyHeader } from 'components/layout';
+
+const EmptyLayout: React.FC = ({ children }) => {
```

Comment:
What's this for?

Reply:
The default layout requires some context providers to already be loaded in order to display. Since this layout is used during initialization this component needs to not have any of those kinds of dependencies.